### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by ThatOneCalculator** [kravetsone/elysia-autoload#11]
Tested on 0.1.0 and 0.1.1

Dev works fine, but build errors.

`build.ts`:

```ts
import { autoload } from "esbuild-plugin-autoload";

await Bun.build({
	entrypoints: ["src/index.ts"],
	outdir: "dist",
	target: "bun",
	sourcemap: "inline",
	minify: true,
	plugins: [
		autoload({
			pattern: "**/*.{ts,tsx,js,jsx,mjs,cjs}",
			directory: "src/routes",
		}),
	],
}).then(console.log);
````

`index.ts`

```ts
const app = new Elysia({
	serve: { reusePort: true },
})
	.use(
		await autoload({
			types: {
				output: "./routes.ts",
				typeName: "Routes",
			},
		}),
	)
```

File structure:

```
.
├── build.ts
├── env.d.ts
├── package.json
├── spawn.ts
├── src
│   ├── index.ts
│   ├── routes
│   │   ├── market
│   │   │   └── create.ts
│   │   ├── signedIn.ts
│   │   ├── signer
│   │   │   ├── approve.ts
│   │   │   └── sessionKey.ts
│   │   ├── timeline
│   │   │   └── [...].ts
│   │   └── user
│   │       ├── follow.ts
│   │       └── [name]
│   │           ├── followers.ts
│   │           ├── following.ts
│   │           └── index.ts
│   └── routes.ts
```

`bun run build.ts`:

```
{
  path: "/home/REDACTED/node_modules/elysia-autoload/dist/index.js",
  namespace: "file",
  loader: "jsx",
}
[ "signedIn.ts", "market/create.ts", "user/follow.ts", "user/[name]/index.ts", "user/[name]/followers.ts",
  "user/[name]/following.ts", "signer/sessionKey.ts", "signer/approve.ts", "timeline/[...].ts"
]
{
  outputs: [],
  success: false,
  logs: [
    74 |                                 ,
                                     ^
error: Expected identifier but found ","
    at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:74:33, 76 |                                 ,
                                     ^
error: Expected identifier but found ","
    at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:76:33, 78 |                                 ,
                                     ^
error: Expected identifier but found ","
    at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:78:33, 80 |                                 ,
                                     ^
error: Expected identifier but found ","
    at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:80:33, 82 |                                 ,
                                     ^
error: Expected identifier but found ","
    at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:82:33, 84 |                                 ,
                                     ^
error: Expected identifier but found ","
    at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:84:33, 86 |                                 ,
                                     ^
error: Expected identifier but found ","
    at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:86:33, 88 |                                 ,
                                     ^
error: Expected identifier but found ","
    at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:88:33
  ],
}
```

**Comment by kravetsone** [comment]
Please add `Build` issues to plugin)

**Comment by kravetsone** [comment]
> Tested on 0.1.0 and 0.1.1
> 
> Dev works fine, but build errors.
> 
> `build.ts`:
> 
> ```ts
> import { autoload } from "esbuild-plugin-autoload";
> 
> await Bun.build({
> 	entrypoints: ["src/index.ts"],
> 	outdir: "dist",
> 	target: "bun",
> 	sourcemap: "inline",
> 	minify: true,
> 	plugins: [
> 		autoload({
> 			pattern: "**/*.{ts,tsx,js,jsx,mjs,cjs}",
> 			directory: "src/routes",
> 		}),
> 	],
> }).then(console.log);
> ```
> 
> `index.ts`
> 
> ```ts
> const app = new Elysia({
> 	serve: { reusePort: true },
> })
> 	.use(
> 		await autoload({
> 			types: {
> 				output: "./routes.ts",
> 				typeName: "Routes",
> 			},
> 		}),
> 	)
> ```
> 
> File structure:
> 
> ```
> .
> ├── build.ts
> ├── env.d.ts
> ├── package.json
> ├── spawn.ts
> ├── src
> │   ├── index.ts
> │   ├── routes
> │   │   ├── market
> │   │   │   └── create.ts
> │   │   ├── signedIn.ts
> │   │   ├── signer
> │   │   │   ├── approve.ts
> │   │   │   └── sessionKey.ts
> │   │   ├── timeline
> │   │   │   └── [...].ts
> │   │   └── user
> │   │       ├── follow.ts
> │   │       └── [name]
> │   │           ├── followers.ts
> │   │           ├── following.ts
> │   │           └── index.ts
> │   └── routes.ts
> ```
> 
> `bun run build.ts`:
> 
> ```
> {
>   path: "/home/REDACTED/node_modules/elysia-autoload/dist/index.js",
>   namespace: "file",
>   loader: "jsx",
> }
> [ "signedIn.ts", "market/create.ts", "user/follow.ts", "user/[name]/index.ts", "user/[name]/followers.ts",
>   "user/[name]/following.ts", "signer/sessionKey.ts", "signer/approve.ts", "timeline/[...].ts"
> ]
> {
>   outputs: [],
>   success: false,
>   logs: [
>     74 |                                 ,
>                                      ^
> error: Expected identifier but found ","
>     at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:74:33, 76 |                                 ,
>                                      ^
> error: Expected identifier but found ","
>     at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:76:33, 78 |                                 ,
>                                      ^
> error: Expected identifier but found ","
>     at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:78:33, 80 |                                 ,
>                                      ^
> error: Expected identifier but found ","
>     at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:80:33, 82 |                                 ,
>                                      ^
> error: Expected identifier but found ","
>     at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:82:33, 84 |                                 ,
>                                      ^
> error: Expected identifier but found ","
>     at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:84:33, 86 |                                 ,
>                                      ^
> error: Expected identifier but found ","
>     at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:86:33, 88 |                                 ,
>                                      ^
> error: Expected identifier but found ","
>     at /home/REDACTED/node_modules/elysia-autoload/dist/index.js:88:33
>   ],
> }
> ```

fixed in [esbuild-plugin-autoload@0.1.2](https://github.com/kravetsone/esbuild-plugin-autoload/releases/tag/v0.1.2)

**Comment by kravetsone** [comment]
Reopen if you still get errors
